### PR TITLE
support mult-terminal wave port (bundle port)

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

File: pyaedt/edb_core/hfss.py [hunk 1]
```diff
@@ -524,6 +525,65 @@ def create_differential_wave_port(
...
+        for p_id, loc in list(zip(primitives_id, points_on_edge)):
+            _, term = self.create_wave_port(
+                p_id,
+                loc,
+                port_name=_port_name,
+                horizontal_extent_factor=horizontal_extent_factor,
+                vertical_extent_factor=vertical_extent_factor,
+                pec_launch_width=pec_launch_width,
+            )
+            _port_name = None
```
Comment (MaxJPRey) [inline pyaedt/edb_core/hfss.py:580]: Do we need to set `_port_name` back to `None` ?

Comment (ring630) [reply pyaedt/edb_core/hfss.py:580]: yes, the terminals must not have the same name. I set it back to None to let create method to determine unique names

Comment (MaxJPRey) [reply pyaedt/edb_core/hfss.py:580]: It seems in the code that _port_name is not used anymore after this line.

Comment (ring630) [reply pyaedt/edb_core/hfss.py:590]: It is used in the for loop
_, term = self.create_wave_port(
                p_id,
                loc,
                port_name=**_port_name,**

Comment (MaxJPRey) [reply pyaedt/edb_core/hfss.py:590]: I see it is used to reset the name of the port_name after the first wave port creation.
Make sense @ring630

**Review by maxcapodi78 — APPROVED** [review]
LGTM

**Pull request merged**